### PR TITLE
MINOR; Some sections are missing in the TOC

### PR DESCRIPTION
**Pull request by jsancio**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by showuon — APPROVED** [review]
Nice catch!

File: docs/toc.html [hunk 1]
```diff
@@ -111,22 +117,28 @@
...
+                        <li><a href="#appvsosflush">Application vs OS Flush Management</a></li>
+                        <li><a href="#linuxflush">Linux Flush Behavior</a></li>
+                        <li><a href="#filesystems">Filesystem Selection</a></li>
+                        <li><a href="#replace_disk">Replace KRaft Controller Disk</a></li>
                     </ul>
+                </li>
                 <li><a href="#monitoring">6.8 Monitoring</a>
                     <ul>
+                        <li><a href="#remote_jmx">Security Considerations for Remote Monitoring using JMX</a></li>
+                        <li><a href="#kraft_monitoring">KRaft Monitoring Metrics</a></li>
```
Comment (showuon) [inline docs/toc.html]: I think we can remove `Metrics` for consistency with the ones below

Comment (jsancio) [reply docs/toc.html]: Yes. Removed metrics from the title.

**Comment by jsancio** [comment]
Thanks for the review @showuon . Merging, cherry-picking to 3.3 and submitting PR to kafka-site.

**Pull request merged**